changelog/pr-2293.md

```diff
@@ -0,0 +1,4 @@
+level: minor
+reference: issue 2293
+---
+The Taskcluster Python client now has an helper function to easily upload artifacts.
```

Comment:
This should probably also be documented in the client's README.

Reply:
Done.